docs/index.md

```diff
@@ -8,7 +8,21 @@ toc: false
 ---
 
 # Why Core-Lib?
-`Core-Lib` was born to make the day-to-day work, The  `"work itself."` easy to master.
+There are a variety of tools and frameworks available to you when building a web application. However, we believe Core-Lib is the best choice for building modern, full-stack web applications.
+
+<b>`Core-Lib` was born to make the day-to-day work, The `"work itself."` easy to master.</b>
+
+### Componentization
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Core-Lib facilitates modular development by breaking down functionalities into smaller, manageable components. This componentization enhances code organization and makes it easier to understand and maintain code.
+
+### Reusability
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Core-Lib helps developers reuse components within the core-lib across different modules, reducing redundancy and saving development time. This reusability fosters consistency and standardization across projects.
+
+### Ease of Test & Debug
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The componentized structure of the core-lib enables developers to test individual components in isolation, allowing for more targeted and effective testing. This leads to higher code quality and fewer bugs in the final product.
+
+### Scalability
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;As the project grows, the core-lib scales with it. Small core-libs can be created for microservices and called inside a main core-lib. New core-libs (features) can be added without significantly impacting existing functionalities, making it easier to maintain and extend the project over time.
```

Comment:
on the how Core-Lib. add a first bullet that say something like 
`Core-Lib` make you project simple as independent code simple code. that is reusable everywhere